### ISSUE TITLE
book of materials "power"

### ISSUE THREAD

**Issue by AquariusPower** [Attnam/ivan#461]
having it on inventory could provide:
- improved info to all items on lists (inventory, drop, equipped etc).
- improved info about NPCs equipped items and golems

improved info is like materials:
- strength
- resistances

something else to add?

having a magnifying lens (a new item) could let fully examine one item (like stethoscopes do with characters), and having the book would show everything related to that item (both materials)

**Comment by iology** [comment]
Looks like spoilers to me. Better discuss on the forum?

The stethoscope always makes me wonder why the monsters are so pleased to do a body check. :P

**Comment by AquariusPower** [comment]
actually, the book already exists, we may find it to be bought at the library or find it at dungeons,
that functionality I propose would just make the book more useful than browsing it, more integrated in the game,

PS.: also, if we could use a filter to just browse it, it would be much better and I think is not difficult to implement.

**Comment by andrewtweber** [comment]
When was this item added? I agree it seems like way too many spoilers.

Isn't there a wizard mode feature to view the material details? If the regular game functionality is starting to match wizard mode functionality then that probably is a sign that it's too powerful

**Comment by fejoa** [comment]
> Isn't there a wizard mode feature to view the material details? If the regular game functionality is starting to match wizard mode functionality then that probably is a sign that it's too powerful

Yeah there's a wizmode feature for this. I agree an in-game feature is kind of redundant, unless there is a material discovery system, in which exotic materials need to be discovered in order to make better use of golems, SoCM etc. 

I think @red-kangaroo added the book. I merged it. It seems innocuous, I don't know what players will gain from knowing about materials, the danger level will still clobber them in the end.

**Comment by AquariusPower** [comment]
@andrewtweber 
may be, the book could be kept but what materials you will gain information about would be limited by some stat, may be intelligence, the same used to change materials. So it would instead be a magic book, where your intelligence limits what you can read.

but I've been using it sometimes and it feels like making decisions about materials is much easier now, removing that feeling of uncertainty about'em.

I added, at crafting, an option to show materials volumes and if an item has a single material, it will also show the material's strength (**I think I should allow str only in wizard mode I guess?**), and that made decisions much easier, and I feel a bit unconfortable about it...

so I propose a system to check an item main material against the other ex.: hit one (main) material item into another (main) material item (whatever it is, from rocks to weapons, armors etc). We can get a message saying what item got scratched (having also a breaking chance if the strengths is too much different like hit a valpurium into iron), and we would know what is the weaker one thru that message. **That would be immersive**!! :)
So, that could be an alternative between having the wizard book and having nothing.

@fejoa 

> exotic materials need to be discovered in order to make better use of golems, SoCM

could you describe this more, sounds interesting :)

>  I don't know what players will gain from knowing about materials

on the crafting system, knowing materials' strength is of utmost importance to not waste time, which w/o enough food can lead to starvation, so that book is of great value, even more if filter capability is added (as currently it is quite annoying to browse it).

**Comment by AquariusPower** [comment]
btw, I think we should only be able to use stethoscope on foes if we are not detected by them. I miss a sneak system also, and that would require vision direction (so we can avoid being seen walking behind foes), and a equipped sound system (where heavy armor making more noise would prevent we getting too near the ones that can hear (if not all)).

**Comment by red-kangaroo** [comment]
I added the book because I don't think it's necessary to hide all info from the player when it can easily be read on the wiki. It does not give more power to the player - it only gives them more choice. They still need the right tool to change stuff into powerful material, but at least they know the material is here and don't always wish for mithril.

> but I've been using it sometimes and it feels like making decisions about materials is much easier now, removing that feeling of uncertainty about'em.

This.

The book is pretty much like the stethoscope - it gives you no power, but let's you learn the game better. Let's you discover more stuff that's otherwise hidden deep in the code.

**Comment by AquariusPower** [comment]
@red-kangaroo ah I get it,
may be if someone doesnt want, then just dont read the book and use the hit-material to see which gets scratched (therefore weaker).
so in short, let user have the option.

**Comment by andrewtweber** [comment]
Similar to polymorph, etc. what if the book only showed them info for materials they have personaly encountered?

**Comment by AquariusPower** [comment]
may be, only the ones that they have examined thru the crafting menu to inspect both of which an item is made of, so that would be a simple trigger/MainFilter, it would end up being like a notebook of experiments instead of a complete reference manual.
